_pages/dev/performance/performance-best-practices.md

```diff
@@ -66,7 +66,7 @@ You can improve the performance of your storefront by following these image form
 Using a CDN can significantly improve the performance of your storefront by distributing your content across multiple servers located around the world, close to your end-users. Here are some best practices for using a CDN:
 
 - Choose a Reliable CDN Provider: Select a CDN provider that offers good performance, reliability, and coverage in the regions where your users are located.
-- Cache Static Assets and Dynamic Prerendered Routes: Configure your CDN to cache static assets such as images, CSS, JavaScript files, fonts and prerendered routes. This reduces the load on your origin server and speeds up content delivery to users.
+- Cache Static Assets and Dynamic Prerendered Routes: Configure your CDN to cache static assets, including images, CSS, JavaScript files, fonts, and prerendered routes. This reduces the load on your origin server and accelerates content delivery to users, generally improving Core Web Vitals metrics such as FCP and LCP.
```

Comment:
`FCP` is a Web Vital, but not a Core one. For more, see https://web.dev/articles/vitals#core-web-vitals

```suggestion
- Cache Static Assets and Dynamic Prerendered Routes: Configure your CDN to cache static assets, including images, CSS, JavaScript files, fonts, and prerendered routes. This reduces the load on your origin server and accelerates content delivery to users, generally improving Core Web Vital metrics such as LCP.
```